hazelcast/src/test/java/com/hazelcast/config/ExternalDataStoreConfigTest.java

```diff
@@ -37,4 +48,48 @@ public void testEqualsAndHashCode() {
                 .suppress(Warning.NONFINAL_FIELDS)
                 .verify();
     }
+
+    @Test
+    public void should_serialize_with_empty_properties() throws IOException {
```

Comment:
non-standard naming scheme

Reply:
we have multiple tests with this scheme which (after some time for adjustment) I've found far more readable ;)